mcp-servers/store_db/helm/templates/postgresql-deployment.yaml

```diff
@@ -0,0 +1,131 @@
+{{- if .Values.postgresql.enabled }}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
We can probably make this a StatefulSet

Reply:
Its only one pod so its does not add value for a demo scenario.. but we can add it in case someone wants to have a multiple pods.